Clinical-Genomics/cg
# Fixes wrong started_at date of analyses in status-db

This PR fixes wrong started_at date of analyses in status-db

How to test:
0. install on rasta stage (add instructions on how to as needed)

1. us
1. on clinical-db, sh db-copy-prod-to-stage.sh trailblazer
1. cg store completed
1. open database to view one of the stored analyses

Expected outcome:
started_at is different from completed_at
